### PR TITLE
docs(showcase): add 74 new showcase sites

### PR DESCRIPTION
**Pull request by slorber**
## Motivation

Add all new showcase submissions above comment https://github.com/facebook/docusaurus/discussions/7826#discussioncomment-3483070

---

FYI we are moving site screenshots to a serverless screenshot service.

```
https://slorber-api-screenshot.netlify.app/<siteUrl>/showcase
```

For example: [https://slorber-api-screenshot.netlify.app/https%3A%2F%2Fdocusaurus.io%2F/showcase](https://slorber-api-screenshot.netlify.app/https%3A%2F%2Fdocusaurus.io%2F/showcase)
Will render:

We know about some limitations of the screenshot service:
- render unwanted announcement bars, cookie banners
- render spinners, does not wait for some elements to load
- fails to render emojis
- respect or not dark mode
- ...

We are going to ship it as is for now, and improve it over time so that all sites in our showcase render nicely

### CODE REVIEW

File: website/src/data/users.tsx [hunk 1]
```diff
@@ -2433,6 +2432,599 @@ const Users: User[] = [
...
+  {
+    title: 'i18n-tools',
+    description: 'CLI to make common operations around i18n files simpler',
+    preview: require('./showcase/7wate-wiki.png'), // TODO
+    website: 'https://jy95.github.io/i18n-tools/',
+    source: 'https://github.com/jy95/i18n-tools/tree/master/website',
+    tags: ['opensource'],
+  },
+  {
+    title: 'Pure.css',
```
Comment (redonkulus) [inline website/src/data/users.tsx:2580]: LGTM

**Comment by slorber** [comment]
All Docusaurus sites submitted have been added in the PR, or rejected with a comment


Ping all submitters:
@vladfrangu @Zamiell @nvh95 @TheCatLady @pkosiec @manuel-rw @jaapstronks @kgajera @Licenser @jspinak @RichKurtzman @yangshun @ViktorWiz @xiaohai-huang @tseijp @thecodinghabits @B4nan @canuto @xiaohuilam @Ouch1978 @RomanKondratyev @ziarparvaiz @ccamel @ariafloat @ainhoa-a @github0null @gustavosbarreto @fishcharlie @anaclumos @jy95 @redonkulus @andrewnicols @kanekotic @nikitavoloboev @lewisrichardson @apisit @hamza1311 @suntudo @dvitanov @jhbeard @DevTomek-pl @vantevo-analytics @danchanka @sgtrueloverokt @alexmercerind @palante @ivishalgandhi @FrancescoXX @yoannfleurydev @wenerme @wafflecoffee @juli1 @webbertakken @charleskorn @WasiqB @rguldener @marcos-muino-garcia @seyfeddin @harinij @nooges @tpascarella @lzwdot @dpimonov @iosifache @gneyal @shoaibsajid1 @arobbins @DrW3RK

**Comment by slorber** [comment]
Ping2 @wafflecoffee @juli1 @webbertakken @charleskorn @WasiqB @rguldener @marcos-muino-garcia @seyfeddin @harinij @nooges @tpascarella @lzwdot @dpimonov @iosifache @gneyal @shoaibsajid1 @arobbins @DrW3RK

**Comment by andrewnicols** [comment]
Perhaps I'm being dense, but how can you tell the state of the filters at the top of the showcase?

I can't tell which of the following (if any) are enabled:

Screen Shot 2022-08-27 at 00 02 47

**Comment by Josh-Cena** [comment]
The green ones are selected. Maybe there isn't enough contrast difference for color-blind folks—I would fix that by adding an icon soon.

**Comment by andrewnicols** [comment]
> The green ones are selected. Maybe there isn't enough contrast difference for color-blind folks—I would fix that by adding an icon soon.

As a colour-blind person, I can confirm. It's only a problem in dark-mode. I expected the buttons to change somehow - perhaps to go from an outlined circle to a filled circle (though that has other issues). Perhaps inverting the colour?

**Comment by Josh-Cena** [comment]
Right. Here's what I made for the TS-ESLint website: https://typescript-eslint.io/rules/

**Comment by andrewnicols** [comment]
> Right. Here's what I made for the TS-ESLint website: https://typescript-eslint.io/rules/

That works well. I can see that ;). Thanks.

File: website/src/data/users.tsx [hunk 2]
```diff
@@ -2433,6 +2432,607 @@ const Users: User[] = [
...
+    tags: ['opensource'],
+  },
+  {
+    title: 'Tremor',
+    description: 'Tremor Event Processing Engine',
+    preview: null,
+    website: 'https://www.tremor.rs/',
+    source: 'https://github.com/tremor-rs/tremor-www/',
+    tags: ['opensource','versioning','large'],
+  },
```
Comment (Licenser) [inline website/src/data/users.tsx:2514]: It seems like our page snuck in twice. That's on me, sorry about that o.O
```suggestion
```

Comment (slorber) [reply website/src/data/users.tsx:2514]: Thanks, it was already added in https://github.com/facebook/docusaurus/pull/6015

**Comment by slorber** [comment]
> Perhaps I'm being dense, but how can you tell the state of the filters at the top of the showcase?

@andrewnicols this is not really a good place to discuss showcase page accessibility issues

I'll keep this in mind, but it would be better to open a dedicated issue, and suggest a proposal that works for you

**Pull request merged**